man/assign.Rd

```diff
@@ -18,6 +18,9 @@
 #            LHS2 = RHS2,
 #            ...), by = ...]
 
+# 3. Multiple columns in place
+# DT[i, .SD = lapply(.SD, fx), by = ..., .SDcols = ...]
```

Comment:
there are not unit test for using `.SD` lhs combined with `i`, and similarly with `by`.